src/common-lib/urls/getDeploymentTestUrls.js

```diff
@@ -71,6 +71,11 @@ function getDeploymentTestUrls() {
     "/teachers/curriculum/english-secondary-aqa/units",
     "/teachers/curriculum/english-secondary-aqa/overview",
     "/teachers/curriculum/english-secondary-aqa/downloads",
+    "/teachers/curriculum/science-secondary-aqa/units",
```

Comment:
For this and the science downloads page, can we have a different exam board than the English example?